IS7024FinalProjectBreweryArc/BreweryArc/Pages/Index.cshtml

```diff
@@ -78,51 +76,47 @@
     <hr />
     <div>
         <table border="1" width="90%">
-            <tr>
-                <th>ID</th>
-                <th>Name</th>
-                <th>Brewery Type</th>
-                <th>Street</th>
-                <th>Address 1st Part</th>
-                <th>Address 2nd Part</th>
-                <th>City</th>
-                <th>State</th>
-                <th>County Province</th>
-                <th>Postal Code</th>
-                <th>Country</th>
-                <th>Longitude</th>
-                <th>Latitude</th>
-                <th>Phone</th>
-                <th>Website Url</th>
-                <th>Update At</th>
-                <th>Created At</th>
-
-            </tr>
-            @{
-                
-                    <tr>
-                        <td>@brewDetail.Id</td>
-                        <td>@brewDetail.Name</td>
-                        <td>@brewDetail.BreweryType</td>
-                        <td>@brewDetail.Street</td>
-                        <td>@brewDetail.Address2</td>
-                        <td>@brewDetail.Address3</td>
-                        <td>@brewDetail.City</td>
-                        <td>@brewDetail.State</td>
-                        <td>@brewDetail.CountyProvince</td>
-                        <td>@brewDetail.PostalCode</td>
-                        <td>@brewDetail.Country</td>
-                        <td>@brewDetail.Longitude</td>
-                        <td>@brewDetail.Latitude</td>
-                        <td>@brewDetail.Phone</td>
-                        <td>@brewDetail.WebsiteUrl</td>
-                        <td>@brewDetail.UpdatedAt</td>
-                        <td>@brewDetail.CreatedAt</td>
-
-                    </tr>
-
-               
-            }
+            <thead>

```

Comment:
I added the table headers into html tag and rest of the table rows in html tag to improve the readability and neatness of the code.
https://www.w3schools.com/tags/tag_thead.asp
https://www.w3schools.com/tags/tag_tbody.asp